WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/BlogDetailHeaderView.swift

```diff
@@ -146,7 +146,7 @@ class BlogDetailHeaderView: UIView {
 
     @objc
     private func siteSwitcherTapped() {
-        delegate?.siteSwitcherTapped()
+        delegate?.siteSwitcherTapped(sourceView: titleView.siteSwitcherButton)
```

Comment:
nitpick: instead of hard-coding to the button instance, we can add an argument to this function and pass it to the delegate.

Reply:
Updated.